contracts/oxo/OxoGame.sol

```diff
@@ -0,0 +1,137 @@
+pragma solidity 0.4.19;
+
+import "./OxoConfig.sol";
+import "./Players.sol";
+import "./GameAction.sol";
+import "./Ownable.sol";
+
+/*In this game one user create a game and set how much he wants to bet.
+Another user should join the game and send equivalent bet number.
+Then game starts*/
+
+contract OxoGame is OxoConfig, GameEntity, GameAction, Players, Ownable {
+
+    event GameCreated(uint gameId, address creator, uint bet);
+    event GameStarted(uint gameId, address player);
+    event UserMadeMove(uint gameId, address player, uint8 index);
+    event GameWon(uint gameId, address winner);
+    event Draw(uint gameId);
+    event MoneyTransferred(uint gameId, address player, uint amount);
+    event GameCanceled(uint gameId);
+    event MoneyWithdrawn(address addr, uint amount);
+
+
+    function createGame() payable public returns (uint){
+        require(msg.value > 0 && msg.value <= maxBet);
+
+
+        Game storage game = createAndGetGame();
+        game.status = GameStatus.CREATED;
+        game.bet = msg.value;
+        game.gameId = games.length - 1;
+        addPlayer(game, msg.sender);
+
+        GameCreated(game.gameId, msg.sender, game.bet);
+        return game.gameId;
+    }
+
+    function joinGame(uint gameId) payable public returns (bool){
+        Game storage game = games[gameId];
+        require(game.status == GameStatus.CREATED);
+        require(msg.value == game.bet);
+        require(game.playerCount < maxPlayerCount);
+
+        addPlayer(game, msg.sender);
+        game.status = GameStatus.STARTED;
+        game.lastMove = msg.sender; // maybe a bit tricky. Game creator must move first
+
+        GameStarted(gameId, msg.sender);
+        return true;
+    }
+
+    function move(uint gameId, uint8 index) public returns (bool){
+        Game storage game = games[gameId];
+        require(index >= 1 && index <= maxFieldNumber);
+        require(game.status == GameStatus.STARTED);
+        require(isPlayer(game, msg.sender)); // if this player has already joined the game
+        require(game.lastMove != msg.sender); // prevent user to make two moves successively
+        require(!isMoved(game, index)); // is move unique
+
+        addMove(game, index, msg.sender);
+        UserMadeMove(gameId, msg.sender, index);
+
+        if (games[gameId].moveCount[msg.sender] <= 2) return true; // if player made only 2 moves he cannot lose or win already
+        bool result = verifyMove(game);
```

Comment:
It  isn't clear from name 'result' what this variable contains. Either it should be draw or win or loose to be true?

Reply:
maybe u right.... result = true if user win
false if not

Reply:
So how do you think you can name this variable?

Reply:
e.g `isWinner`